model/accounts_mgmt/v1/quota_cost_type.model

```diff
@@ -19,5 +19,5 @@ struct QuotaCost {
 	QuotaID string
 	Allowed integer
 	Consumed integer
-	RelatedResources []interface
+	RelatedResources []RelatedResources
```

Comment:
As attribute is an array and the name of the attribute is in plural, I'd expect the name of the type of the items to be in singular: `RelatedResource`.